spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, :type => :controller do
+
+  describe '#create' do
+    #### User factory girl to generate these - refactor opportunity
+    let!(:user) { User.create!(username: 'hello', email: 'hello@hello.com', password: "hello") }
+    let!(:idea) { Idea.create!(title: 'hellohello', description: 'hellohellohello', creator: user) }
+    before { allow(controller).to receive(:current_user) { user } }
+    let!(:comment) { idea.comments.create!(author: user, text: 'hellohellohello') }
+
+    it 'creates a new comment from Idea' do
+      expect do
+        post :create, {
+          comment: {
+            commentable_type: 'Idea', commentable_id: idea.id, text: 'this is a great idea'
+          }
+        }
+     end.to change{Comment.count}.by 1
+    end
+
+    # TEST REMOVED FOR POLYMORPHIC COMMENTS ON COMMENTS FEATURE
```

Comment:
Same as below, commented code can be found in git history (if you're practicing good git hygiene), so no use polluting files with unnecessary commented code.